src/calendar.cpp

```diff
@@ -1,12 +1,17 @@
 #include <cmath>
 #include <sstream>
+#include <limits>
+
 #include "calendar.h"
 #include "output.h"
 #include "options.h"
 #include "translations.h"
 #include "game.h"
 #include "debug.h"
 
+// Divided by 100 to prevent overflowing when converted to turns
+const calendar calendar::INDEFINITELY_LONG( int( std::numeric_limits<int>::max() / 100 ) );
```

Comment:
Why the cast to int, shouldn't it already be an int? Also, the comment is not clear. The calendar constructor takes turns as input, there is no conversion there, did you mean moves (as in `Creature::moves`)?